Puzzlepart/prosjektportalen365
# Feature/799 prepoulate helplist

### Your checklist for this pull request


### Description

Prepopulate helplist with content to navigation elements in portfolio and project.

### How to test

1. Apply the helplist
2. Check that the list exist in the configuration site
3. See and click on the help function on the top right corner of every navigation element.


### Relevant issues (if applicable)
closes #799 

💔Thank you!
